Update build

nightscape opened the pull request.

nightscape committed:
Update SBT, Scala and dependencies

tindzk commented:
Sorry for the delay! For some reason, I did not receive an e-mail notification.

Your changes look fine, but the Travis build is failing. It appears that uPickle's API has changed and the Scala Native plug-in was not found.

nightscape commented:
I updated the code for the new uPickle version and removed the Scala Native stuff because of missing transitive dependencies.
Travis is ok now.
Sorry for submitting this PR in a failing state initially.

tindzk commented:
Looks great! Thanks for your contribution.
